installer/lib/phx_new/single.ex

```diff
@@ -67,6 +67,17 @@ defmodule Phx.New.Single do
     {:eex, "phx_test/views/page_view_test.exs",              :project, "test/:lib_web_name/views/page_view_test.exs"},
   ]
 
+  template :live, [
+    {:eex, "phx_live/controllers/page_controller.ex",        :project, "lib/:lib_web_name/controllers/page_controller.ex"},
+    {:eex, "phx_web/templates/layout/app.html.eex",          :project, "lib/:lib_web_name/templates/layout/app.html.eex"},
+    {:eex, "phx_web/templates/page/index.html.eex",          :project, "lib/:lib_web_name/templates/page/index.html.eex"},
+    {:eex, "phx_web/views/layout_view.ex",                   :project, "lib/:lib_web_name/views/layout_view.ex"},
+    {:eex, "phx_live/live/page_live_view.ex",                :project, "lib/:lib_web_name/live/page_live_view.ex"},
+    {:eex, "phx_test/controllers/page_controller_test.exs",  :project, "test/:lib_web_name/controllers/page_controller_test.exs"},
+    {:eex, "phx_test/views/layout_view_test.exs",            :project, "test/:lib_web_name/views/layout_view_test.exs"},
+    {:eex, "phx_test/live/page_live_view_test.exs",         :project, "test/:lib_web_name/live/page_live_view_test.exs"},
+  ]
+
```

Comment:
There is definitely overlap with `template :html`.  I wasn't sure how best to manage conflicts with the same name for the _sample_ liveview where the `phx_live/controllers/page_controller.ex` is different.  I also wasn't sure if we wanted to manage all cases

| live? | html? | Description |
| --- | --- | --- |
| F | F | No HTML / Views at all |
| F | T | Yes HTML, no LiveView |
| T | F | Only show a LiveView example |
| T | T |  Show both the LiveView AND the RegularView |

Reply:
Live implies HTML. If Live is given, we won't generate a HTML. So the fourth row in your table is not necessary.

Reply:
Alternatively, we don't show a Live example here and leave it for later (in a possible phx.gen.live generator). Especially because if we have a Live example, we don't need a controller.

Reply:
I have been playing around with generated examples, and I am going to look at having live just add a snippet within the HTML so that both can co-exist as it does provide a nice view of live and non-live code.

I’ll look separately at the generator, which might make more sense in the long run as “—live” might become the default